### ISSUE TITLE
Catalytic annotations on TFIIH subunits & PAINT propagations

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#2894]
Human TFIIH is characterized to consist of 10 subunits in 2 subcomplexes. The complex is known to have several activities, including ATPase and protein kinase.
See: https://www.genenames.org/data/genegroup/#!/group/1745 & PMID:29664212

The catalytic subunits are characterized as:
- CKD7 -> kinase activity
- ERCC2 & ERCC3 -> ATPase activity

Curation of PMID:9852112 (1998) by SWIS added the following MF (IDA) annotations:

- contributes_to protein kinase activity (GO:0004672):
	- P18074 (ERCC2)
	- P19447 (ERCC3)
	- P32780 (GTF2H1)
	- Q13888 (GTF2H2)
	- Q13889 (GTF2H3)
	- Q92759 (GTF2H4)
- contributes_to RNA polymerase II CTD heptapeptide repeat kinase activity (GO:0008353):
	- P18074 (ERCC2)
	- P19447 (ERCC3)
	- P32780 (GTF2H1)
	- Q13888 (GTF2H2)
	- Q13889 (GTF2H3)
	- Q92759 (GTF2H4)
	- P51948 (MNAT1)
	- P51946 (CCNH)
- contributes_to DNA-dependent ATPase activity (GO:0008094):
	- P18074 (ERCC2)
	- P19447 (ERCC3)
	- P32780 (GTF2H1)
	- Q13888 (GTF2H2)
	- Q13889 (GTF2H3)
	- Q92759 (GTF2H4)
	- P51948 (MNAT1)
	- P51946 (CCNH)

According to the current 'contributes_to' guidelines, I think all these (except the ERCC2 & ERCC3 ATPase annotations?) need to be removed.

The main problem here is that these annotations are seeding propagation of the MF terms via several PAINT nodes, including:
PANTHER:PTN000158239 (P18074)
PANTHER:PTN000138598 (P19447)
PANTHER:PTN000304482 (P32780)
PANTHER:PTN000302934 (Q13889)
PANTHER:PTN000322344 (Q92759)
PANTHER:PTN000002807 (P51946)

@sylvainpoux  - can you review these annotations?
@pgaudet  - pending on the above, you might want to directly edit the GO terms on the PAINT nodes?

**Comment by sjm41** [comment]
@hattrill

**Comment by sylvainpoux** [comment]
Hi,
I can check these annotations
I however admit some frustration...
It would be good if GO could anticipate when they decide to change some annotation policy and propose semi-automatic modifications.

We have been told to use the "contributes_to" for many years, and cannot manually review all annotations that are not compliant with the current rule

Not your fault in any case, and I will check these annotations 

Cheers

Sylvain

**Comment by sjm41** [comment]
Thanks Sylvain - much appreciated.
Completely agree about the revised 'contributes_to' rules - they need to more widely 'advertised' amongst curators as well as some thought applied about a systematic retrofit.
I'm only finding (and reporting) inconsistencies in annotations when they affect annotations to Drosophila enzymes via PAINT nodes - these things keep cropping up as we continue our systematic review of Drosophila enzyme annotations....

**Comment by sylvainpoux** [comment]
Sure
anyway, I deleted these annotations
Cheers
Sylvain

**Comment by sjm41** [comment]
Thanks Sylvain.
@hattrill / @pgaudet - do either of you know if there are any thoughts/plans about a more systematic (perhaps semi-automated) retrofit of old "contributes_to" annotations that are inconsistent with the revised rules?

**Comment by hattrill** [comment]
@sjm41 I've suggested that we could propose a retrofit of contributes_to at the GOC meeting in spring (a virtual meeting now). @bmeldal I think we could get a reasonable proposal by May and assess the extent of the problem and if there are any computational methods that will speed this task.

**Comment by bmeldal** [comment]
Happy to work with you guys on this. I think that is the main Complex WG outstanding issue.

**Comment by RLovering** [comment]
UCL has used contributes_to in nearly 500 annotations https://www.ebi.ac.uk/QuickGO/annotations?qualifier=contributes_to&assignedBy=ARUK-UCL,BHF-UCL,HGNC-UCL,ParkinsonsUK-UCL 
annotating almost 200 unique gene symbols
If you want to use ours a test set please do
Ruth

**Comment by hattrill** [comment]
Adding link to original ticket #1650

**Comment by hattrill** [comment]
I'll make a separate ticket for a retrofit proposal.

**Comment by sjm41** [comment]
Thanks Helen. Sylvain has addressed the specific issue described on this ticket so I'll close this one now.
